app/app/src/main/java/com/globant/calculator/android/mvp/presenter/CalculatorPresenter.java

```diff
@@ -13,4 +21,53 @@ public CalculatorPresenter(CalculatorModel model, CalculatorView view) {
         this.view = view;
     }
 
+    public void onClearButtonPress() {
+        model.clear();
+        view.showOperationPressed(NUMBER_ZERO);
+        view.showResult(ZERO_INT);
+    }
+
+    public void onAddButtonPressed(String symbol) {
+        model.setOperator(symbol);
+        view.showOperationPressed(PLUS);
+    }
+
+    public void onMultiplyButtonPressed(String symbol) {
+        model.setOperator(symbol);
+        view.showOperationPressed(MULTIPLY);
+    }
+
+    public void onSubtractionButtonPressed(String symbol) {
+        model.setOperator(symbol);
+        view.showOperationPressed(MINUS);
+    }
+
+    public void onDivideButtonPressed(String symbol) {
+        model.setOperator(symbol);
+        view.showOperationPressed(DIVIDE);
+    }
+
+    public void onEqualsButtonPressed(String symbol) {
+
```

Comment:
If this method will be added some funcionality, put a "//wille be added when implements the equals button" comment, Becouse I do not know it will be or not used. Thanks

Reply:
added the comment